contrib/playwright/docker-compose.yml

```diff
@@ -58,7 +58,7 @@ services:
   #     timeout: 5s
 
   ui-test:
-    image: "mcr.microsoft.com/playwright:${PLAYWRIGHT_TAG:-v1.48.1}"
+    image: "mcr.microsoft.com/playwright:${PLAYWRIGHT_TAG:-v1.44.1}"
```

Comment:
I'm not sure downgrading this version is a good idea.

It might cause inconsistencies with the version used during CI too - they always have to match else browser versions used for testing will differ